floris/type_dec.py

```diff
@@ -59,7 +59,7 @@ def floris_array_converter(data: Iterable) -> np.ndarray:
         raise TypeError(e.args[0] + f". Data given: {data}")
     return a
 
-def floris_numeric_dict_converter(data: dict) -> dict:
+def floris_numeric_dict_converter(data: dict, allow_strings=False) -> dict:
```

Comment:
Consider an "allow_any" flag instead of specifically allowing strings. Also, consider not using this function if you want to convert a dictionary that isn't all numbers. And also there's a new argument added but the docstring isn't updated.

Reply:
Yeah, I'm now also realizing it may make more sense to load the data stored in the string ahead of time, which would avoid this issue. I'm going to see if that'll work